docker/php-test-stepup/Dockerfile

```diff
@@ -11,13 +11,19 @@ RUN apt-get update && apt-get install -y \
     python \
     zip \
     libpng-dev \
+    wget\
     && docker-php-ext-install pdo_mysql exif gd \
     ## APCu
     && pecl install apcu \
     && pecl install apcu_bc-1.0.3 \
     && docker-php-ext-enable apcu --ini-name 10-docker-php-ext-apcu.ini \
     && docker-php-ext-enable apc --ini-name 20-docker-php-ext-apc.ini
 
+RUN apt-get install -y libgmp-dev re2c libmhash-dev libmcrypt-dev file
+RUN ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/local/include/
+RUN docker-php-ext-configure gmp
+RUN docker-php-ext-install gmp
```

Comment:
Idem for these lines. They can probably be included on the 'Install dependencies' `RUN` section on lines 9..20